bamboo/compiler_tests/test_compiler.py

```diff
@@ -107,17 +109,19 @@ def build_skeleton(dir_name, compiler, mpi_lib, debug, should_log):
     output_file_name = '%s/bamboo/compiler_tests/output/%s_%s_build_output.txt' % (dir_name, compiler_underscored, build_type)
     error_file_name = '%s/bamboo/compiler_tests/error/%s_%s_build_error.txt' % (dir_name, compiler_underscored, build_type)
     compiler = compiler.replace('@', '-')
-    mpi_lib = mpi_lib.replace('@', '-')
+    #mpi_lib = mpi_lib.replace('@', '-')
     cluster = re.sub('[0-9]+', '', subprocess.check_output('hostname'.split()).strip())
     # For reference:
     # x86_64 <=> catalyst, pascal, quartz, surface
     # ppc64le <=> ray
     architecture = subprocess.check_output('uname -m'.split()).strip()
     if cluster == 'ray':
         architecture += '_gpu_cuda-9.2.64_cudnn-7.0'
-    elif cluster in ['pascal', 'surface']:
-        architecture += "_gpu"
-    os.chdir('%s/bamboo/compiler_tests/builds/%s_%s_%s_%s_openblas_%s/build' % (dir_name, cluster, compiler, architecture, mpi_lib, build_type))
+    elif cluster == 'pascal':
+        architecture += '_gpu_cuda-9.1.85_cudnn-7.1'
+    elif cluster == 'surface':
+        architecture += '_gpu'
+    os.chdir('%s/bamboo/compiler_tests/builds/%s_%s_%s/build' % (dir_name, cluster, compiler, build_type))
```

Comment:
Do we even need `architecture` anymore then? It looks like it's not used now.

Reply:
Probably not. I will remove it before merging. There are probably several commented out or otherwise not used blocks that I will get rid of pre-merge

Reply:
As discussed, I guess we'll comment out the `architecture` block.